weekly-mission(week2)/src/constants/validate.js

```diff
@@ -0,0 +1,6 @@
+import REGEX from "./regex.js";
+
+export const isValid = {
+  email: (email) => REGEX.EMAIL.test(email),
+  password: (password) => REGEX.PASSWORD.test(password)
+};
```

Comment:
이런식으로 객체로 작성되려면 변수명이 validator(검증을 하는 도구)와 같은 느낌이 괜찮을 것 같은데요.

`검증을 하는 도구(validtor)를 가져와서 email에 대해 검증하겠다.` 요런 느낌이 될 것 같아요!
```suggestion
export const validator = {
  email: (email) => REGEX.EMAIL.test(email),
  password: (password) => REGEX.PASSWORD.test(password)
};
```

Reply:
해당 파일에는 변수가 1개만 있는데 다른 상수 파일과 다르게 `named export`가 되어 있어서 컨벤션을 맞춰주는게 중요할 것 같아요!